dcmnet/apps/movescu.cc

```diff
@@ -1468,11 +1472,11 @@ static OFCondition storeSCP(
       /* remove file */
       if (!opt_ignore)
       {
-        if (strcmp(imageFileName, NULL_DEVICE_NAME) != 0) OFStandard::deleteFile(imageFileName);
+        if (strcmp(imageFileName, NULL_DEVICE_NAME) != 0) OFStandard::deleteFile(ofname.c_str());
       }
 #ifdef _WIN32
     } else if (opt_ignore) {
-        if (strcmp(imageFileName, NULL_DEVICE_NAME) != 0) OFStandard::deleteFile(imageFileName); // delete the temporary file
+        if (strcmp(imageFileName, NULL_DEVICE_NAME) != 0) OFStandard::deleteFile(ofname.c_str()); // delete the temporary file
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

We observed that in these codes. `Multiline blocks` are missing. It is considered best practice to enclose all `multiline blocks` including if else blocks, loops, and function bodies within curly braces. To mitigate this issue, it serves to clearly define the scope of the block and avoid logic errors by grouping all related statements together visually.

Reply:
Good morning. I was preserving the style as it was before the changes. Per your request/suggestions, I went ahead and added the scope braces.